hs/src/Reach/Parser.hs

```diff
@@ -142,61 +160,96 @@ instance Pretty JSBundle where
           ("// " <> viaShow rs) :
           map (pretty . ppShow) jms
 
-gatherDeps_fc :: SrcLoc -> IORef JSBundleMap -> JSFromClause -> IO JSFromClause
-gatherDeps_fc at fmr (JSFromClause ab aa s) = do
-  s_abs <- gatherDeps_file GatherNotTop (srcloc_at "import from" (tp ab) at) fmr $ trimQuotes s
+gatherDeps_fc
```

Comment:
Is there a number of columns after which you'd make an exception? I generally try to prevent horizontal scrolling or exceeding 100 if I can help it but occasionally that doesn't make sense.

This line would be 115 characters if I flattened it.

Wrapped lines and crazy long lines are hostile to readability IMHO, both in the editor and on GitHub/wherever + rendered docs.